rllib/evaluation/tests/test_episode.py

```diff
@@ -0,0 +1,143 @@
+import ray
```

Comment:
nice! we just have to add this into `rllib/BUILD` to actually get this to run

Reply:
@richardliaw It has been added to rllib/BUILD and is passing